DataConnectors/Forcepoint DLP.json

```diff
@@ -0,0 +1,85 @@
+{
+    "id": "Forcepoint_DLP",
+    "title": "Forcepopint DLP",
+    "publisher": "Forcepoint",
+    "descriptionMarkdown": "The Forcepoint DLP connector allows you to automatically export DLP incident data from Forcepoint DLP into Azure Sentinel in real-time. This enriches visibility into user activities and data loss incidents, enables further correlation with data from Azure workloads and other feeds, and improves monitoring capability with Workbooks inside Azure Sentinel.",
+    "graphQueries": [
+        {
+            "metricName": "Total data received",
+            "legend": "ForcepointDLPEvents_CL",
+            "baseQuery": "ForcepointDLPEvents_CL"
+        }
+    ],
+    "sampleQueries": [
+        {
+            "description" : "Rules triggered in the last three days",
+            "query": "ForcepointDLPEvents_CL\n | where TimeGenerated > ago(3d)\n | summarize count(RuleName_1_s) by RuleName_1_s, SourceIpV4_s\n | render barchart"
+        },
+        {
+            "description" : "Rules triggered over time (90 days)",
+            "query": "ForcepointDLPEvents_CL\n | where TimeGenerated > ago(90d)\n | sort by CreatedAt_t asc nulls last\n | summarize count(RuleName_1_s)  by  CreatedAt_t, RuleName_1_s\n | render linechart"
+        },
+        {
+            "description" : "Count of High, Medium and Low rules triggered over 90 days",
+            "query": "ForcepointDLPEvents_CL\n | where TimeGenerated > ago(90d)\n | sort by CreatedAt_t asc nulls last\n | summarize count(Severity_s)  by  CreatedAt_t, Severity_s\n | render barchart"
+        }
+    ],
+    "dataTypes": [
+        {
+            "name": "ForcepointDLPEvents_CL",
+            "lastDataReceivedQuery": "ForcepointDLPEvents_CL\n            | summarize Time = max(TimeGenerated)\n            | where isnotempty(Time)"
+        }
+    ],
+    "connectivityCriterias": [
+        {
+            "type": "IsConnectedQuery",
+            "value": [
+                "ForcepointDLPEvents_CL\n            | summarize LastLogReceived = max(TimeGenerated)\n            | project IsConnected = LastLogReceived > ago(30d)"
+            ]
+        }
+    ],
+    "availability": {
+        "status": 1
+    },
+    "permissions": {
+        "resourceProvider": [
+            {
+                "provider": "Microsoft.OperationalInsights/workspaces",
+                "permissionsDisplayText": "read and write permissions are required.",
+                "providerDisplayName": "Workspace",
+                "scope": "Workspace",
+                "requiredPermissions": {
+                    "write": true,
+                    "read": true,
+                    "delete": true
+                }
+            }
+        ]
+    },
+    "instructionSteps": [
+        {
+            "title": "",
+            "description": "1. Step by step instructions to use this connector with Forcepoint DLP at this link: \n https://support.forcepoint.com/DocumentsDisplayed?version=8.7&name=Forcepoint%20DLP",
```

Comment:
1. Restructure the text in the instruction step and removed bullet as there’s just one item
"Follow step-by-step instructions provided at [Forcepoint DLP documentation](https://linkname) to configure this connector. \n"
2. Also when you are referring to product documentation links like https://support.forcepoint.com/DocumentsDisplayed?version=8.7&name=Forcepoint%20DLP , please share redirection links that you can manage on your side. That way we don’t need to re-update and re-deploy the data connector when the link changes as the redirection link in the data connectors will be static while the contents of the static link can be managed by you and updated when it changes.
3. This link https://support.forcepoint.com/DocumentsDisplayed?version=8.7&name=Forcepoint%20DLP goes to a general page and there are no instructions – is this the right link or do you plan to have detailed instructions to configure this connection in that link later – if later ETA of that? Refer to point #2 above on the link in this connector itself.